client/my-sites/exporter/exporter.jsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import Gridicon from 'components/gridicon';
+import Button from 'components/forms/form-button';
+import AdvancedOptions from 'my-sites/exporter/advanced-options';
+
+export default React.createClass( {
+	displayName: 'Exporter',
+
+	render: function() {
+		return (
+			<div className="section-export">
+				<CompactCard>
+					<header>
+						<Button
+							className="exporter__export-button"
+							disabled={ false }
+							isPrimary={ true }
+						>
+							{ this.translate( 'Export' ) }
+						</Button>
+						<h1 className="exporter__title">
+							{ this.translate( 'Download an Export File' ) }
+						</h1>
+					</header>
+					<a href="#" onClick={ this.props.toggleAdvancedSettings }>
+						<Gridicon
+							icon={ this.props.advancedSettings.isVisible ? 'chevron-up' : 'chevron-down' }
+							size={ 16 } />
+						{ this.translate( 'Advanced Export Settings' ) }
+					</a>
+				</CompactCard>
+
+				{
+					this.props.advancedSettings.isVisible &&
+					<AdvancedOptions
```

Comment:
I need to run, but I'd like to expand more on this point later: "Being more granular and making a best effort to send non-complex props makes it much easier to achieve pure components, improves reusability, and clearly identifies the needs of the component." There are several occurrences in the post editor where reliance on passing large complex objects has made building new components much more difficult than it needs to be.